test/new-e2e/tests/otel/utils/pipelines_utils.go

```diff
@@ -499,11 +494,36 @@ func getLoadBalancingSpans(t assert.TestingT, traces []*aggregator.TracePayload)
 			}
 		}
 	}
-	assert.Equal(t, 12, spans)
+	require.Equal(t, 12, spans)
 	return spanMap
 }
 
-func getLoadBalancingMetrics(t assert.TestingT, metrics []*aggregator.MetricSeries) map[string][]map[string]string {
+func getLoadBalancingLogs(c require.TestingT, s OTelTestSuite, service string) {
+	logs1, err := s.Env().FakeIntake.Client().FilterLogs(service, fakeintake.WithMessageMatching(log1Body))
+	require.NoError(c, err)
+	require.NotEmpty(c, logs1)
```

Comment:
should this be `require.Len(c, logs1, 1)` ?

Reply:
I tried this at first but since there are lots of logs being generated, `FilterLogs` will always return more than 1 log. We only need the first log here so it's sufficient to check `NotEmpty`.